src/realm/group_shared.cpp

```diff
@@ -463,11 +463,13 @@ struct alignas(8) SharedGroup::SharedInfo {
 
     uint64_t number_of_versions; // Offset 32
 
-    /// True (1) if there is a sync client present. It is an error to start a
-    /// sync client if another one is present. If the sync client crashes and
-    /// leaves the flag set, the session will need to be restarted (lock file
-    /// reinitialized) before a new sync client can be started.
-    uint8_t sync_client_present = 0; // Offset 40
+    /// True (1) if there is a sync agent present (a session participant acting
+    /// as sync client). It is an error to have a session with more than one
+    /// sync agent. The purpose of this flag is to prevent that from ever
+    /// happening. If the sync agent crashes and leaves the flag set, the
+    /// session will need to be restarted (lock file reinitialized) before a new
+    /// sync agent can be started.
+    uint8_t sync_agent_present = 0; // Offset 40
```

Comment:
Rnaming `SharedInfo::sync_client_present` to `SharedInfo::sync_agent_present` for terminological consistency. This does not affect the on-disk format.